src/Draco.Debugger.Tests/TestDebugSession.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections.Immutable;
+using System.Runtime.CompilerServices;
+using Basic.Reference.Assemblies;
+using Draco.Compiler.Api;
+using Draco.Compiler.Api.Syntax;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Draco.Debugger.Tests;
+
+public class TestDebugSession
```

Comment:
Sealed?

Reply:
It's unit test code, I usually don't care about it's maintainability as long as it's do it's job.
I'll add it.